pilot/platform/cloudfoundry/controller.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudfoundry
+
+import (
+	"reflect"
+	"time"
+
+	"golang.org/x/net/context"
+
+	copilotapi "code.cloudfoundry.org/copilot/api"
```

Comment:
our typical import pattern is:
```
go packages
<new line>
non-istio packages
<new line>
istio package
```

Reply:
fixed in https://github.com/istio/istio/pull/2022/commits/ee17f605f08afccc85eadf6e973ee25b9df3968b